applications/ShapeOptimizationApplication/tests/sensitivity_verification_process_test/run_sensitivity_verification_in_design_space.py

```diff
@@ -170,4 +173,15 @@ def OutputModelPart( output_mdpa, output_filename, nodal_results ):
             line_to_write += '%.6E' % Decimal(str(fd_gradient_y)) + ",\t"
             line_to_write += '%.6E' % Decimal(str(fd_gradient_z)) + ",\t"
             line_to_write += str(time.ctime()) + "\n"
-            open_file.write(line_to_write)
\ No newline at end of file
+            open_file.write(line_to_write)
+
+# ==============================================================================
+# Postprocessing
+# ==============================================================================
+if delete_results_afterwards:
+    kratos_utilities.DeleteDirectoryIfExisting("__pycache__")
+    kratos_utilities.DeleteFileIfExisting("sensitivity_verification_process_test.post.lst")
+    kratos_utilities.DeleteFileIfExisting(results_filename)
+    kratos_utilities.DeleteFileIfExisting("structure.post.bin")
+    kratos_utilities.DeleteFileIfExisting("reference_part.post.bin")
+    kratos_utilities.DeleteFileIfExisting("perturbed_part.post.bin")
```

Comment:
why do you delete them?
In the regular testing routine I would not write them, since they are not used and take up valueable computation time
In case you want to look at them (e.g. while setting up a test or debugging) then you wouldn't want to delete them :)

Reply:
That is true. In fact, the purpose of those scripts is to look at the produced output. So once you use it, you would deactivate the deletion of the files. The purpose of the test, however, is to guarantee that the process is running so that the script remains "plug and play".